tests/unit/test_timestamp.py

```diff
@@ -0,0 +1,71 @@
+import unittest
```

Comment:
____________________________________________________________________________________________ [pylint] tests/unit/test_timestamp.py ____________________________________________________________________________________________
C: 35, 0: Line too long (126/100) (line-too-long)
C: 53, 0: Line too long (126/100) (line-too-long)
C: 55, 0: Line too long (102/100) (line-too-long)
C: 64, 0: Line too long (126/100) (line-too-long)
C: 69, 0: Line too long (123/100) (line-too-long)
C: 71, 0: Trailing newlines (trailing-newlines)
R: 33, 4: Method could be a function (no-self-use)
R: 42, 4: Method could be a function (no-self-use)
R: 51, 4: Method could be a function (no-self-use)
R: 62, 4: Method could be a function (no-self-use)
C:  3, 0: standard import "from unittest import mock, TestCase" should be placed before "import urllib3" (wrong-import-order)